themes/custom/ts_wrin/templates/ts_custom/maintenance-page.html.twig

```diff
@@ -0,0 +1,73 @@
+{% if is_node %}
+  {{ drupal_block('local_tasks_block') }}
+{% endif %}
+{% if page.wrin_custom.wrinfilterstatus %}
+  {% include 'wrin-filter-status' %}
+{% endif %}
+<div id="tray-nav-wrapper" style="height: 100vh;">
+  <div id="tray-nav-canvas">
+
+    <div class="mobile-menu-target">
+      <div class="ts-mobile-menu__wrapper">
+        {% if page.hamburger_nav %}
+          {{ page.hamburger_nav }}
+        {% endif %}
+      </div>
+    </div>
+
+    {# Start - Off-canvas elements: Filter, Search, Mega-menu #}
+    <div id="tray-nav">
+      {% if page.wrin_custom %}
+        {{ page.wrin_custom }}
+      {% endif %}
+    </div>
+    {# End - Off-canvas elements #}
+
+    {% if page.admin %}
+      {{ page.admin }}
+    {% endif %}
+
+    {% if page.content %}
+      {% if is_node %}
+        {{page.content}}
+      {% else %}
+      <div class="landing-page__content container margin-top-md margin-bottom-md">
+        <div class="content__inner" style="text-align: center;">
+          {{ drupal_block('local_tasks_block') }}
+          <img class="logo-color" src="/sites/default/files/nav_logo_color.svg" alt="Home" style="margin: 40px auto;"> {{page.content}}
```

Comment:
We can't count on this file existing, or on this page being displayed in English, so can you pull in the branding block? Probably something like `{{ drupal_block('system_branding_block') }}`. Test it out on the China branch to be sure it works. This is what I'm getting here: https://pr-170-wri-china.pantheonsite.io/ with maintenance mode turned on.
<img width="864" alt="Screen Shot 2022-03-10 at 9 22 30 AM" src="https://user-images.githubusercontent.com/999525/157719651-4685b92f-80f3-4310-b478-25151cd34cf0.png">

Reply:
Fixed and noted the translation points for WRI China in https://github.com/wri/WRIN/issues/338#issuecomment-1064666942

![Screen Shot 2022-03-10 at 5 19 53 PM](https://user-images.githubusercontent.com/1668324/157783338-25258dbe-d327-4af6-a93a-8f0a7231258d.png)